src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -785,6 +905,16 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     return REDISMODULE_ERR;
   }
 
+  if (RedisModule_CreateCommand(ctx, "ray.set_add", SetAdd_RedisCommand, "write", 0, 0,
```

Comment:
Should be `"write pubsub"`, right?

Reply:
Seems you are right. But why `RAY.TABLE_APPEND` is `write`?

Reply:
According to https://redis.io/topics/modules-api-ref , `RAY.TABLE_APPEND` should be also set to "write pubsub". However, it looks like these commands work fine without `pubsub`... The document does not explain how it will use `strflags`, but we should keep it right.

Reply:
I've updated `RAY.TABLE_APPEND`, `RAY.SET_ADD` and `RAY.SET_REMOVE` to `write pubsub` flag.